src/stirling/source_connectors/socket_tracer/uprobe_symaddrs.cc

```diff
@@ -142,19 +142,33 @@ StatusOr<std::string> InferHTTP2SymAddrVendorPrefix(ElfReader* elf_reader) {
   return vendor_prefix;
 }
 
+std::optional<int64_t> ResolveSymbolWithEachGoPrefix(ElfReader* elf_reader,
```

Comment:
Can you add a comment above this function explaining the symbol naming change (essentially what was in your PR description mentions)?